src/providers/test-submission/test-submission.ts

```diff
@@ -13,9 +14,12 @@ export class TestSubmissionProvider {
     ) {}
 
   submitTests = (data: StandardCarTestCATBSchema[]): Observable<any> => {
+
+    const compressedData = gzipSync(JSON.stringify(data));
```

Comment:
Quick unit test to cover this?

Reply:
I'm pulling this out to a new PR as it's not working how i'd like.